packages/light.js/src/rpc/other/post.ts

```diff
@@ -55,22 +57,10 @@ export function post$ (tx: Tx, options: PostOptions = {}) {
         const gas = await api.eth.estimateGas(tx);
         observer.next({ estimated: gas });
       }
-      const signerRequestId = await api.parity.postTransaction(tx);
-      observer.next({ requested: signerRequestId });
-      const transactionHash = await api.pollMethod(
-        'parity_checkRequest',
-        signerRequestId
-      );
-      if (tx.condition) {
-        observer.next({ signed: transactionHash, schedule: tx.condition });
-      } else {
-        observer.next({ signed: transactionHash });
 
-        const receipt = await getTransactionReceipt(transactionHash, api);
-        observer.next({ confirmed: receipt });
-      }
+      const signedTransaction = await api.personal.signTransaction(tx, passphrase);
+      postRaw$(signedTransaction.raw).subscribe(observer);
```

Comment:
Since this is a breaking change, what do you think of putting `observer.next({ signed: ... })` here

Reply:
sounds good!